scripts/globals/mobskills/rushing_drub.lua

```diff
@@ -7,23 +7,24 @@
 --  Range: Melee
 --  Notes: Only used by Mamool Ja with staves.
 ---------------------------------------------
-
-require("scripts/globals/settings")
 require("scripts/globals/status")
 require("scripts/globals/monstertpmoves")
-
 ---------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
-    return 0
+    if (mob:getMainJob() == dsp.job.BLM or mob:getMainJob() == dsp.job.WHM) and mob:AnimationSub() == 0 then
```

Comment:
what if I'm coded as a war/whm with stave?

I think there is a battlefield mob used when the AC is taken coded such a way. I could be wrong but its more flexible to just have a skill list for stave users than to tie it to main job ID, since SE has been known to do odd things like that.

Like the a certain gigas monk with the great axe, or a warrior behemoth casting meteor.

Reply:
can make another pull separating mamool jobs at a later date